src/forms/inputs/radio-group.js

```diff
@@ -89,9 +90,18 @@ const defaultProps = {
   radioInputProps: {},
 }
 
-function RadioGroupLegend({ label, name }) {
-  if (label === false) return null
-  return <legend>{label || convertNameToLabel(name)}</legend>
+function RadioGroupLegend({ label, name, required, requiredIndicator, hint }) {
+  return (
+    <legend className={classnames({ 'visually-hidden': label === false })}>
```

Comment:
I think we'll also need to add a note about this in the migration guide. If the user is coming from an older project, they might not have `visually-hidden` styles included in their scss set up (or they might have it applied by another name).